app/views/items/index.html.erb

```diff
@@ -5,7 +5,7 @@
       <h3 class="category"> <%= category.title %></h3>
           <% category.items.each do |item| %>
 
-            <div class="Column" style="text-align: center;">
+            <div class="<%= item.title %>" style="text-align: center;">
```

Comment:
might be worth putting the column div back in place outside of current div to keep bootstrap in order